kong-frontend-config.json

```diff
@@ -35,6 +35,24 @@
       ],
       "minimumReleaseAge": "10 days"
     },
+    {
+      "automerge": true,
+      "groupName": "all non-major peerDependencies with stable version",
+      "groupSlug": "all-minor-patch-peer",
+      "matchCurrentVersion": "!/^0/",
+      "matchDepTypes": [
+        "peerDependencies"
+      ],
+      "matchPackagePatterns": [
+        "*"
+      ],
+      "matchUpdateTypes": [
+        "minor",
+        "patch"
+      ],
+      "rangeStrategy": "widen",
+      "minimumReleaseAge": "10 days"
```

Comment:
@baumac The only new properties here are `matchDepTypes` and `rangeStrategy` - I'm assuming I still need the rest to enforce our stratgegies